### PR TITLE
Add configuration by constants to javascript code

### PR DESCRIPTION
**Pull request by janveselynet**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Default ignored files
```
Comment (michalbrauner) [inline .idea/.gitignore]: toto je tady nejaky zapomenuty ne? :)

Comment (janveselynet) [reply .idea/.gitignore]: Dik, to mi uniklo :-)

File: src-php/Javascript/ConstantCollection.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+use function implode;
+
+final class ConstantCollection implements UiComponent
+{
+    /**
+     * @var array
+     */
+    private $constants;
+
+
```
Comment (michalbrauner) [inline src-php/Javascript/ConstantCollection.php]: radek navic

File: src-php/Javascript/Constant.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var UiComponent
+     */
+    private $value;
+
+
```
Comment (michalbrauner) [inline src-php/Javascript/Constant.php]: radek navic

File: src-php/Javascript/ConstantCollection.php [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+use Assert\Assertion;
+use BrandEmbassy\Components\UiComponent;
+use function array_map;
+use function implode;
+
+final class ConstantCollection implements UiComponent
+{
+    /**
+     * @var array
```
Comment (michalbrauner) [inline src-php/Javascript/ConstantCollection.php]: `UiComponent[]`

File: src-php/Javascript/Value/StringValue.php [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types = 1);
+
+namespace BrandEmbassy\Components\Javascript\Value;
+
+use BrandEmbassy\Components\StringEscaper;
+use BrandEmbassy\Components\UiComponent;
+use function sprintf;
+
+final class StringValue implements UiComponent
```
Comment (michalbrauner) [inline src-php/Javascript/Value/StringValue.php]: co to dat pod neaky interface `PrimitiveTypeUiComponent` ?

**Pull request merged**